pkg/clustermesh/mcsapi/serviceimport_controller.go

```diff
@@ -294,7 +331,9 @@ func checkConflictExport(orderedSvcExports []*mcsapitypes.MCSAPIServiceSpec) str
 			if conflictCount > 0 {
 				return fmt.Sprintf(
 					"Conflicting %s. %d/%d clusters disagree. Using \"%s\" from oldest service export in cluster \"%s\".",
-					getterStruct.name, conflictCount, clusterCount, oldestField, orderedSvcExports[0].Cluster,
+					fieldStruct.name, conflictCount, clusterCount,
```

Comment:
Nit. (although unrelated to this PR) `conflictCount` is always guaranteed to be greater than zero, and the if check could be dropped.